code-server/run.sh

```diff
@@ -70,4 +70,24 @@ for extension in "$${EXTENSIONLIST[@]}"; do
   fi
 done
 
+if [ "${AUTO_INSTALL_EXTENSIONS}" = true ]; then
+  WORKSPACE_DIR="$HOME/"
+  if [ -n "${FOLDER}" ]; then
+    WORKSPACE_DIR="${FOLDER}"
+  fi
+
+  if ! command -v jq > /dev/null; then
+    echo "jq is required to install extensions from a workspace file."
```

Comment:
Not sure if this is overkill, but we could the Node bundled with code-server. 

```
extensions=$("$INSTALL_PREFIX/lib/node" -e "require(\"$WORKSPACE_DIR/.vscode/extensions.json\").recommendations.forEach((r) => console.log(r))")
```

Reply:
Interesting. I do like `jq`, but if we know the path to `node` and it should always be present, then it makes sense.